gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -1136,34 +1140,41 @@ def main(args):
                         "monoallelic": dt_ht.info.monoallelic,
                         "only_het": dt_ht.info.only_het,
                     }
-                # TODO: Should we be adding filtering info to the joint HT?
+
```

Comment:
After L1133, you need to add: 
```commandline
                    for_joint_validation=(dt == "joint"),
```

Reply:
Because it was set to `True` by default, or maybe we should set default to `False`.

Reply:
I think `for_joint_validation` is supposed to be True for all of these. I was a bit confused by it at first, which is why I added the data_type change originally, but I think I have it right now. @mike-w-wilson might know best though

Reply:
No, this is needed when we're validate `exomes` and `genomes` release HT, because it's set to True, we need it to be false, so we have other INFO fields included.

Reply:
oh, I see what you are saying, but what we need is `for_joint_validation=for_joint,`

Reply:
if `dt` is "exomes" or "genomes", but `for_joint` (`data_type=="joint") we want `for_joint_validation` to still be True

Reply:
Yeah, this was how we can differentiate if were validating exomes and genomes for release or validating them within the joint HT.

Reply:
Yes, thanks, my brain is not so sharp today...